src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -873,8 +875,11 @@ public Response deleteProject(
                 }
 
                 LOGGER.info("Project " + project + " deletion request by " + super.getPrincipal().getName());
+
                 try {
-                    qm.recursivelyDelete(project, true);
+                    final Handle jdbiHandle = createLocalJdbi(qm).open();
```

Comment:
The `Handle` still needs closing to avoid resource leakage.

```suggestion
                try (final Handle jdbiHandle = createLocalJdbi(qm).open()) {
```

Reply:
Correct 👍🏼